assets/js/modules/analytics-4/datastore/api.js

```diff
@@ -52,17 +50,7 @@ const baseSelectors = {
 			}
 		}
```

Comment:
We should probably also consider the GA4 `getAccountSummaries` selector here - if that randomly failed, we also have breakage likely due to an API change.

Also, why isn't this selector using `createRegistrySelector` and actually checking for something like `hasFinishedResolution`? Regarding the errors, it would be more appropriate to introduce new `getErrorsForSelector( selectorName )` and `getErrorsForAction( actionName )` selectors via `createErrorStore` which we could then use here - but that would increase scope further, so I think the manual `state.errors` checks here are okay.